docs/source/usage/execution.rst

```diff
@@ -32,14 +32,14 @@ Additionally, further information can be given to ``PyExperimenter``:
 
 - ``experiment_configuration_file_path``: The path of the :ref:`experiment configuration file <experiment_configuration_file>`. Default: ``config/experiment_configuration.cfg``.
 - ``database_credential_file_path``: The path of the :ref:`database credential file <database_credential_file>`. Default: ``config/database_credentials.cfg``
+- ``use_ssh_tunnel``: Specifies if a SSH tunnel will be used to connect to the database. Default: ``False``. If ``use_ssh_tunnel`` is set to ``True``, creating a ``PyExperimenter`` will also open an ssh tunnel, which should be closed manually. For more information check :ref:`close_ssh_tunnel <close_ssh_tunnel>`.
```

Comment:
Reference to the database config doc, as it requires a specific ssh part

Reply:
I do not understand